src/@types/roleType.ts

```diff
@@ -0,0 +1,4 @@
+export interface roleTypes {
+	id?: string;
+	roleName?: string;
```

Comment:
check well we have two file, one is "roletype.ts" and "roleTypes.ts" simplify them in one.